arangodb-net-standard/TransactionApi/TransactionApiClient.cs

```diff
@@ -49,6 +49,7 @@ public TransactionApiClient(IApiClientTransport client, IApiClientSerialization
         /// <typeparam name="T">Type to use for deserializing the object returned by the transaction function.</typeparam>
         /// <param name="body">Object containing information to submit in the POST transaction request.</param>
         /// <returns>Response from ArangoDB after processing the request.</returns>
+        /// <remarks>https://www.arangodb.com/docs/stable/http/transaction-stream-transaction.html</remarks>
```

Comment:
Currently only JS-Transactions are supported, but surely we will add support for stream transaction at some point. Maybe link to the parent page instead? https://www.arangodb.com/docs/stable/http/transaction.html

Reply:
Also, can you move the `remarks` tag right after `summary`? Trivial but I think it helps to read it that way.